### PR TITLE
Remove old experiment page UI

### PR DESCRIPTION
**Pull request by bryce-fitzsimons**
### Features and Changes

- Remove legacy experiment page
- Add banner to help users find settings

image

- Closes **(add link to issue here)**

### CODE REVIEW

File: packages/front-end/components/Experiment/SinglePage.tsx [hunk 1]
```diff
@@ -194,6 +195,11 @@ export default function SinglePage({
   editPhases,
   editPhase,
 }: Props) {
+  const [hideNewExperimentHelp, setHideNewExperimentHelp] = useLocalStorage(
+    `experiment-page__hide-new-experiment-help`,
+    false
```
Comment (jdorn) [inline packages/front-end/components/Experiment/SinglePage.tsx:200]: Should we be smarter about the default value?

```ts
let alreadyUsingNewPage = false;
try {
  if (window.localStorage.getItem("single-page-new-ui-v1") === "true") {
    alreadyUsingNewPage = true;
  }
} catch(e) {
  // Ignore localStorage errors
}

const [hideNewExperimentHelp, setHideNewExperimentHelp] = useLocalStorage(
    `experiment-page__hide-new-experiment-help`,
    // Hide if the user has already been using the new page
    alreadyUsingNewPage
);
```

File: packages/front-end/components/Experiment/SinglePage.tsx [hunk 2]
```diff
@@ -784,21 +790,50 @@ export default function SinglePage({
...
         </div>
 
+        {!hideNewExperimentHelp ? (
+          <div
+            className="d-flex justify-content-end small mt-1"
+            style={{ marginBottom: -10 }}
+          >
+            <div
+              className="text-gray py-1 rounded text-center d-flex align-items-center justify-content-center"
+              style={{ backgroundColor: "#f993", width: 530 }}
```
Comment (jdorn) [inline packages/front-end/components/Experiment/SinglePage.tsx:814]: Not sure about a red background. Makes it look like an error.

Comment (bryce-fitzsimons) [reply packages/front-end/components/Experiment/SinglePage.tsx]: Sure, let's go with periwinkle
image

File: packages/front-end/components/Experiment/SinglePage.tsx (hunk 1 above)
Comment (bryce-fitzsimons) [reply packages/front-end/components/Experiment/SinglePage.tsx]: incorporated your suggestion

**Review by jdorn — APPROVED** [review]

**Pull request merged**